setup/tinyHIPPOstart

```diff
@@ -14,21 +14,21 @@ pip3 install scapy
 pip3 install dataclasses
 pip3 install requests
 
-# set config path environment variable
-export IOT_IDS_CONFIGPATH='/etc/tinyHIPPO/config.json'
+# set install path environment variable
+export TINYHIPPO_INSTALL_PATH='/etc/tinyHIPPO/'
 
 # Move the dashboard files
 # TODO: this is a hack to get tinyHIPPO running for the midterm progress report
-cp -r /etc/tinyHIPPO/src/dashboard/luasrc/controller/ids-priv /usr/lib/lua/luci/controller
-cp -r /etc/tinyHIPPO/src/dashboard/luasrc/view/ids-priv /usr/lib/lua/luci/view
+cp -r $TINYHIPPO_INSTALL_PATH/src/dashboard/luasrc/controller/ids-priv /usr/lib/lua/luci/controller
+cp -r $TINYHIPPO_INSTALL_PATH/dashboard/luasrc/view/ids-priv /usr/lib/lua/luci/view
```

Comment:
This copy command throws an error. Should be $TINYHIPP_INSTALL_PATH/src/dashboard/.....